### PR TITLE
Updated changelog to account for recent changes

### PR DESCRIPTION
**Pull request by IvanSavenko**
To be merged before creation of `beta` branch. Will update PR to include all PR's merged by that time

### CODE REVIEW

**Comment by SoundSSGood** [comment]
Cheat code "vcmiscrolls" added.

File: ChangeLog.md [hunk 1]
```diff
@@ -110,6 +138,8 @@
 * Added support for banning objects per zones
 * Added support for customizing objects frequency, value, and count per zone
 * Fixed values of Pandora Boxes with creatures to be in line with H3:SoD
+* Added sealed zone types, for entirely unpassable zones.
+* It is now possible connect two zones with multiple connections of the same type
```
Comment (DjWarmonger) [inline ChangeLog.md:142]:
```suggestion
It is now possible connect two zones with multiple connections of same or different types

**Comment by DjWarmonger** [comment]
I'd also suggest splitting battle AI and adventure AI changes.

**Comment by dydzio0614** [comment]
I would add in AI section: "Fixed BattleAI hyperfocusing unreachable targets while missing all kinds of another attack opportunities"

**Pull request merged**